service/worker/deletenamespace/reclaimresources/workflow.go

```diff
@@ -128,7 +133,15 @@ func ReclaimResourcesWorkflow(ctx workflow.Context, params ReclaimResourcesParam
 		return result, err
 	}
 
-	// Step 2. Delete namespace.
+	// Step 2. Sleep before deleting namespace from database.
+	if params.NamespaceDeleteDelay > 0 {
+		err = workflow.Sleep(ctx, params.NamespaceDeleteDelay)
```

Comment:
workflow.Sleep on zero or negative is fine, can remove the condition to simplify the code